packages/common-all/src/error.ts

```diff
@@ -183,3 +183,22 @@ export function assertInvalidState(msg: string): never {
     message: msg,
   });
 }
+
+/** Utility class for helping to correctly construct common errors. */
+export class ErrorFactory {
+  static createUnexpectedEventError({ event }: { event: any }): DendronError {
```

Comment:
Event is `any` since `HistoryEvent` is defined in `engine-server` package and errors.ts file is in `common-all` package.